hack/build-images.sh

```diff
@@ -70,49 +70,46 @@ os::provision::install-sdn "${OS_ROOT}" "${imagedir}" "${OS_ROOT}/images/node"
 mkdir -p images/node/conf/
 cp -pf "${OS_ROOT}/contrib/systemd/openshift-sdn-ovs.conf" images/node/conf/
 
-# builds an image and tags it two ways - with latest, and with the release tag
-function image {
+# image-build is wrapped to allow output to be captured
+function image-build {
   local tag=$1
   local dir=$2
-  local out
-  out="$( mktemp )"
-
-  set +e
-  (
-    set -euo pipefail
-
-    local dest="${tag}"
-    if [[ ! "${tag}" == *":"* ]]; then
-      dest="${tag}:latest"
-    fi
+  local dest="${tag}"
+  if [[ ! "${tag}" == *":"* ]]; then
+    dest="${tag}:latest"
+  fi
 
-    local STARTTIME=$(date +%s)
+  local STARTTIME=$(date +%s)
 
-    # build the image
-    if ! os::build::image "${dir}" "${dest}"; then
-      os::log::warn "Retrying build once"
-      os::build::image "${dir}" "${dest}"
-    fi
+  # build the image
+  if ! os::build::image "${dir}" "${dest}"; then
+    os::log::warn "Retrying build once"
+    os::build::image "${dir}" "${dest}"
+  fi
 
-    # tag to release commit unless we specified a hardcoded tag
-    if [[ ! "${tag}" == *":"* ]]; then
-      docker tag "${dest}" "${tag}:${OS_RELEASE_COMMIT}"
-    fi
-    # ensure the temporary contents are cleaned up
-    git clean -fdx "${dir}"
+  # tag to release commit unless we specified a hardcoded tag
+  if [[ ! "${tag}" == *":"* ]]; then
+    docker tag "${dest}" "${tag}:${OS_RELEASE_COMMIT}"
+  fi
+  # ensure the temporary contents are cleaned up
+  git clean -fdx "${dir}"
 
-    local ENDTIME=$(date +%s); echo "Finished in $(($ENDTIME - $STARTTIME)) seconds"
-  ) > "${out}" 2>&1
-  local rc=$?
-  set -e
+  local ENDTIME=$(date +%s); echo "Finished in $(($ENDTIME - $STARTTIME)) seconds"
```

Comment:
At some point I want to turn on that trap by default for everyone, and I don't see a compelling reason for you to make my life harder :)